src/actions/condition/Condition.ts

```diff
@@ -0,0 +1,49 @@
+import Action from "../Action";
+import {IConditionAction} from "./IConditionAction";
+import Param from "../../parameters/Param";
+
+const CONDITIONAL_OPERATORS: Record<string, string> = {
+  "=": 'eq',
+  "!=": 'ne',
+  "<": 'lt',
+  ">": 'gt',
+  "<=": 'lte',
+  ">=": 'gte',
+  "&&": 'and',
+  "||": 'or',
+  "*": "mul",
+  "/": "div",
+  "+": "add",
+  "-": "sub",
+  "^": "pow"
+};
+
+class ConditionAction extends Action implements IConditionAction {
+  constructor(expression: string) {
+    super();
+
+    this.addParam(new Param('if', this.replaceExpression(expression)));
+
+  }
+
+  replaceExpression(expression: string) {
+    const resultExpression = expression
+      .split(' ')
+      .map((val) => CONDITIONAL_OPERATORS[val] || val)
+      .join('_');
+
+
+    return resultExpression;
+  }
+}
+
+/**
+ * @param key
```

Comment:
Please add the missing JSDoc (if we already know what they are from PHP 2.0)

If we don't know what they should be, please open an internal issue for future work to ensure the docstrings are added.

Reply:
updated with what is know and opened a ticket for unknowns